phpunit.xml

```diff
@@ -1,15 +1,15 @@
+<?xml version="1.0"?>
```

Comment:
Does this config works for PHPUnit <10? The way I understood it, it doesn't.

Reply:
It may or may not but the framework is only using a single version in this PR (~10).